VultisigApp/VultisigApp/Views/Keysign/KeysignDiscoveryView.swift

```diff
@@ -169,6 +169,7 @@ struct KeysignDiscoveryView: View {
                 .onAppear {
                     if participantDiscovery.peersFound.count == 1 && participantDiscovery.peersFound.first == peer {
                         handleSelection(peer)
+                        startKeysign()
```

Comment:
For this one , we need to make sure that we already have more than the threshold before we kick off keysign. 

can you make sure `viewModel.selections.count < (vault.getThreshold() + 1)` meet?

Reply:
Hum... Let me double-check.

Reply:
Done